src/main/java/com/google/api/gax/grpc/ApiCallable.java

```diff
@@ -119,7 +119,7 @@ public ResponseT call(RequestT request) {
    * @param context {@link com.google.api.gax.grpc.CallContext} to make the call with
    * @param observer Observer to interact with the result
    */
-  public void asyncCall(CallContext<RequestT> context, final StreamObserver<ResponseT> observer) {
+  private void asyncCall(CallContext<RequestT> context, final StreamObserver<ResponseT> observer) {
```

Comment:
You missed the other asyncCall. I think we should delete them instead of making them private, otherwise we'll have unused dead code.